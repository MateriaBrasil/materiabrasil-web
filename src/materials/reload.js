import React from 'react'
import Redirect from 'react-router-dom/Redirect'

export default props => () => {
  const { match } = props
  const { id } = match.params

  return <Redirect to={`/materials/${id}?reload=true`} />
}
