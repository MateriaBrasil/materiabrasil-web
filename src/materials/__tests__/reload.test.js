import React from 'react'
import renderer from 'react-test-renderer'

import reload from '../reload'

jest.mock('react-router-dom/Redirect', () => props => (
  <div {...props}>Redirect</div>
))

it('renders correctly', () => {
  const Component = reload({ match: { params: { id: 1234 } } })
  const tree = renderer.create(<Component bar="foo" />).toJSON()
  expect(tree).toMatchSnapshot()
})
