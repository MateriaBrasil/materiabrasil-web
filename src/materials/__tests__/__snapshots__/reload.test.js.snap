// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  to="/materials/1234?reload=true"
>
  Redirect
</div>
`;
